### PR TITLE
Add create_tables.cql script

### PR DESCRIPTION
**Pull request by octonato**
Relates to https://github.com/akka/akka-platform-guide/pull/220

### CODE REVIEW

File: src/main/g8/ddl-scripts/create_tables.cql [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+-- This CQL script will create the keyspace and all tables needed for the this sample. 
+-- It includes the messages and snapshot tables (write-side) and the projection offset_store table (read-side).
+
+-- NOTE: the keyspace as created here is probably not what you need in a production environment. 
+-- This is good enough for local development though. 
+
+CREATE KEYSPACE IF NOT EXISTS shoppingcart
+  WITH REPLICATION = { 'class' : 'SimpleStrategy','replication_factor':1 };
+
+USE shoppingcart;
```
Comment (patriknw) [inline src/main/g8/ddl-scripts/create_tables.cql]: this should be derived from the name given to g8. Let's use
```
"$name;format="word-only,lowercase"$"
```

that becomes `shoppingcartservice`

Comment (octonato) [reply src/main/g8/ddl-scripts/create_tables.cql]: correct

Comment (octonato) [reply src/main/g8/ddl-scripts/create_tables.cql]: Done, without the quotes around it.

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
Then one more iteration in the tutorial example to change the keyspace from `shoppingcart` to `shoppingcartservice`

**Pull request merged**